django/db/models/sql/query.py

```diff
@@ -489,6 +489,9 @@ def has_filters(self):
     def has_results(self, using):
         q = self.clone()
         if not q.distinct:
+            if q.group_by is True:
```

Comment:
Yes, because it can also be "truthy" by containing a list of fields. True means all fields.